### DISCUSSION TITLE
where to put "rootfs.tar.xz" on Windows?

### DISCUSSION THREAD

**Discussion by gerroon** [podman-desktop/podman-desktop#5372] [Q&A] [answered]
Hi

I am behind proxy, so I set up the proxy in PD settings but for some reason it is not able to use fot these downloads. I can download it manually but I do not know where it needs to go?

```
Error: Command execution failed with exit code 125 Command execution failed with exit code 125 Error: head request failed: https://github.com/containers/podman-wsl-fedora/releases/latest/download/rootfs.tar.xz: Head "https://github.com/containers/podman-wsl-fedora/releases/latest/download/rootfs.tar.xz": proxyconnect tcp: dial tcp: lookup http: no such host

```

**Comment by gerroon** [comment]
Basically these settings do not work

![image](https://github.com/containers/podman-desktop/assets/8519469/5d57adc0-466e-4849-a308-7585cdad783a)

**Comment by benoitf** [comment, marked as answer]
@gerroon you should download the airgap installer/ binary ( it includes the .tar.xz file) from the website podman-desktop.io

**Reply by gerroon** [reply]
Thank you, that worked for the initial pod setup, but I am unable to login t the running container. I am not sure if this is tied to the internet issue

```
 Error: crun: executable file `/bin/sh` not found in $PATH: No such file or directory: OCI runtime attempted to invoke a command that was not found

```
